crates/bevy_ui/src/layout/mod.rs

```diff
@@ -57,7 +82,7 @@ impl fmt::Debug for UiSurface {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         f.debug_struct("UiSurface")
             .field("entity_to_taffy", &self.entity_to_taffy)
-            .field("window_nodes", &self.window_nodes)
+            .field("window_nodes", &self.root_nodes)
```

Comment:
Isn't it better to rename the field too, since this is only for debug print anyway so we care little about back compatibility?

```suggestion
            .field("root_nodes", &self.root_nodes)
```